_episodes/09-connected-components.md

```diff
@@ -3,11 +3,269 @@ title: "Connected Component Analysis"
 teaching: ??
 exercises: ??
 questions:
-- "How to describe objects in images."
+- "How to describe objects in images in terms of numbers."
```

Comment:
`"in terms of numbers"` is a bit too generic I think. 
Not sure what the best alternative is though.
Maybe reformulate the question to
`"How to count the number of distinct objets in images."`
Or something similar?

Reply:
thank you @constantinpape for having a look!

What do you think about "How to describe object properties with numbers?"

Reply:
> "How to describe object properties with numbers?"

Hmm, I think I would split the question into two parts, because we actually do two different things here:
- extracting objects via connected components.
- compute features for these objects via regionprops.

Suggestion:
"How to extract separate objects from an image and describe these objects in terms of numbers." (or maybe replace "in terms of numbers" with "quantitatively").

Reply:
thx for this wonderful suggestion @constantinpape - I like it :)